### PR TITLE
Old mapping syntax has been deprecated

### PR DESCRIPTION
**Pull request by ohadbitt**
Old mapping syntax has been deprecated

### CODE REVIEW

**Comment by PRMerger9** [comment]
@ohadbitt : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by orspod** [comment]
@vladikbr since this has been deprecated - should we totally remove it from the documentation? If so, we need to make sure all queries are updated with the new syntax.

**Comment by vladikbr** [comment]
> @vladikbr since this has been deprecated - should we totally remove it from the documentation? If so, we need to make sure all queries are updated with the new syntax.

@orspod I agree. This is completely deprecated and should be  eliminated from all documentation and examples.

**Comment by orspod** [comment]
> > @vladikbr since this has been deprecated - should we totally remove it from the documentation? If so, we need to make sure all queries are updated with the new syntax.
> 
> @orspod I agree. This is completely deprecated and should be eliminated from all documentation and examples.

@vladikbr which developer can assist with this task? I assume any docs dealing with mapping will have the old format and will need to be updated.
@ohadbitt please remove any references to the old format from this doc. Tag me for review when complete

**Comment by PRMerger17** [comment]
@ohadbitt : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger6** [comment]
@ohadbitt : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger10** [comment]
@ohadbitt : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger12** [comment]
@ohadbitt : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger13** [comment]
@ohadbitt : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger16** [comment]
@ohadbitt : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger12** [comment]
@ohadbitt : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger14** [comment]
@ohadbitt : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger7** [comment]
@ohadbitt : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger18** [comment]
@ohadbitt : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger19** [comment]
@ohadbitt : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger19** [comment]
@ohadbitt : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger20** [comment]
@ohadbitt : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger20** [comment]
@ohadbitt : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger6** [comment]
@ohadbitt : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger6** [comment]
@ohadbitt : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger6** [comment]
@ohadbitt : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger7** [comment]
@ohadbitt : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger8** [comment]
@ohadbitt : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger9** [comment]
@ohadbitt : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger10** [comment]
@ohadbitt : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger12** [comment]
@ohadbitt : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: data-explorer/kusto/api/netfx/kusto-ingest-queued-ingest-sample.md [hunk 1]
```diff
@@ -101,7 +113,10 @@ static void Main(string[] args)
         // Usually the recommended level is IngestionReportLevel.FailuresOnly
         ingestProps.ReportLevel = IngestionReportLevel.FailuresAndSuccesses;
         ingestProps.ReportMethod = IngestionReportMethod.Queue;
-        ingestProps.JSONMappingReference = mappingName;
+        ingestProps..IngestionMapping = new IngestionMapping()
```
Comment (vladikbr) [inline data-explorer/kusto/api/netfx/kusto-ingest-queued-ingest-sample.md:116]: double dot (ingestProps..IngestionMapping)

**Comment by PRMerger14** [comment]
@ohadbitt : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by orspod** [comment]
@ohadbitt - please remove the mention of the old format and examples from this file: data-explorer/kusto/management/mappings.md.
According to @vladikbr request, we should remove mention of old deprecated format.

**Comment by orspod** [comment]
@kerend and @lucygoldbergmicrosoft please review the changes on your docs.

File: data-explorer/ingest-json-formats.md [hunk 2]
```diff
@@ -113,13 +113,14 @@ Use C# to ingest data in raw JSON format.
...
+            Data.Ingestion.IngestionMappingKind.Json,
             tableName,
             tableMapping,
-            new[]
-            {
-                new JsonColumnMapping {ColumnName = "Events", JsonPath = "$"},
-            });
+            new[] {
+            new ColumnMapping {ColumnName = "Events", Properties = new Dictionary<string, string>() {
+                {"path","$.records"} }
```
Comment (kerend) [inline data-explorer/ingest-json-formats.md:122]: Path changed here, is it intended?

Comment (ohadbitt) [reply data-explorer/ingest-json-formats.md:122]: no

**Comment by PRMerger12** [comment]
@ohadbitt : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: data-explorer/kusto/api/netfx/kusto-ingest-client-reference.md [hunk 3]
```diff
@@ -4,21 +4,20 @@ description: This article describes Kusto.Ingest client interfaces and factory c
...
 author: orspod
 ms.author: orspodek
-ms.reviewer: rkarlin
+ms.reviewer: ohbitton
 ms.service: data-explorer
 ms.topic: reference
-ms.date: 03/24/2020
+ms.date: 05/19/2020
 ---
 # Kusto.Ingest client interfaces and factory classes
```
Comment (orspod) [inline data-explorer/kusto/api/netfx/kusto-ingest-client-reference.md:12]:
```suggestion
# Kusto.Ingest client interfaces and classes
```

**Comment by PRMerger17** [comment]
@orspod : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by orspod** [comment]
#sign-off

**Pull request merged**